templates/url_base/new.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}New UrlBase{% endblock %}
+
+{% block body %}
+    <h1>Create new UrlBase</h1>
```

Comment:
1. To nie jest dobry opis pola dla użytkownika - on nie wie co to jest urlBase
2. Zadbaj, żeby wszystkie teksty były przygotowane do tłumaczenia